third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -825,7 +827,7 @@ const char* gOnNewWindowScript = R""""(
   window.__RECORD_REPLAY__ = window.top.__RECORD_REPLAY__;
   window.__RECORD_REPLAY_ARGUMENTS__ = window.top.__RECORD_REPLAY_ARGUMENTS__;
 
-  for (g of window.top.__RECORD_REPLAY_GLOBALS__) {
+  for (const g of window.top.__RECORD_REPLAY_GLOBALS__) {
```

Comment:
Note: My [previous concern](https://github.com/replayio/chromium/pull/1122#discussion_r1482983362) was not addressed -

* We also need to pass down `__RECORD_REPLAY_GLOBALS__` itself.
* In fact, I'd vote to only pass down `__RECORD_REPLAY_GLOBALS__`, and just force any user of this API to go through that, rather than adding random globals into a global context that is shared with arbitrary user JS.